flow-server/src/main/java/com/vaadin/flow/server/communication/StreamReceiverHandler.java

```diff
@@ -237,7 +238,9 @@ private boolean handleMultipartFileUploadFromParts(VaadinSession session,
                 getLogger().warn(limitInfoStr, "Request size",
                         "getRequestSizeMax");
             } else if (e instanceof UploadFileSizeLimitExceededException) {
-                getLogger().warn(limitInfoStr, "File size", "getFileSizeMax");
+                UploadFileSizeLimitExceededException fileSizeException = (UploadFileSizeLimitExceededException) e;
```

Comment:
nit: can maybe avoid cast
```suggestion
} else if (e instanceof UploadFileSizeLimitExceededException fileSizeException) {
```